### PR TITLE
W-16312987-alt-text-trust-day-1.10-kt

### PR DESCRIPTION
**Pull request by kevintroller**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/index-vm-bare-metal.adoc [hunk 1]
```diff
@@ -28,11 +28,11 @@ Anypoint Runtime Fabric on VMs / Bare Metal supports development and production
 
 The development configuration is intended for testing only. It requires at least one controller and two worker nodes. The controller node runs the internal load balancer and agents used to connect to Anypoint Platform. Communication between the agent and Anypoint Platform is always outbound. Multiple replicas of application can run across workers.
 
-image::architecture-development.png[]
+image::architecture-development.png["Diagram of a network development architecture of MuleSoft control plane with controllers, such as internal load balancer, agent, and workers"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/index-vm-bare-metal.adoc]:
```suggestion
image::architecture-development.png["Diagram of a network development architecture of the MuleSoft control plane with controllers, such as an internal load balancer, agent, and workers"]
```

File: modules/ROOT/pages/index-vm-bare-metal.adoc [hunk 2]
```diff
@@ -28,11 +28,11 @@ Anypoint Runtime Fabric on VMs / Bare Metal supports development and production
...
 
 === Production Configuration
 
-image::architecture-production.png[]
+image::architecture-production.png["Diagram of a Mule development with control plane, controllers, such as internal load balancer, agent, workers, and TCP load balancer"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/index-vm-bare-metal.adoc]:
```suggestion
image::architecture-production.png["Diagram of a Mule development with the control plane, controllers, such as an internal load balancer, agent, workers, and TCP load balancer"]
```

File: modules/ROOT/pages/index-vm-bare-metal.adoc [hunk 3]
```diff
@@ -85,7 +85,7 @@ The successful operation of Anypoint Runtime Fabric on VMs / Bare Metal is a sha
 
 The following image illustrates MuleSoft and customer responsibilities for on-premises Runtime Fabric instances:
 
-image::runtimefabricresponsibility.png[Runtime Fabric Shared Responsiblities]
+image::runtimefabricresponsibility.png[Runtime Fabric Shared Responsibilities]
```
Comment (valkyrie69) [inline modules/ROOT/pages/index-vm-bare-metal.adoc]: Generally, don't just repeat the name of the image in the alt text. Is this decorative or something? Then [""]

File: modules/ROOT/pages/install-create-rtf-arm.adoc [hunk 4]
```diff
@@ -25,7 +25,7 @@ If you do not see the *Runtime Fabrics* tab in Runtime Manager as described belo
 [IMPORTANT]
 After creating the Runtime Fabric, Runtime Manager displays the corresponding activation data. Record this data to use later when installing running the Runtime Fabric on VMs / Bare Metal installer.
 
-image::runtime-fabric-arm.png[]
+image::runtime-fabric-arm.png["Runtime Fabric instance installation steps in Runtime Manager UI"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/install-create-rtf-arm.adoc]:
```suggestion
image::runtime-fabric-arm.png["Runtime Fabric instance installation steps in the Runtime Manager UI"]
```

File: modules/ROOT/pages/install-prereqs.adoc [hunk 5]
```diff
@@ -310,7 +310,7 @@ The controller VM that is the leader for installation performs the following act
 A minimum of 6 nodes are required for a production configuration of Runtime Fabric on VMs/ Bare Metal, as shown in
 the following diagram.
 
-image::architecture-production.png[]
+image::architecture-production.png["Diagram of a Mule development with control plane, controllers, such as internal load balancer, agent, workers, and TCP load balancer"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/install-prereqs.adoc]:
```suggestion
image::architecture-production.png["Diagram of a Mule development with the control plane, controllers, such as an internal load balancer, agent, workers, and TCP load balancer"]
```

File: modules/ROOT/pages/install-prereqs.adoc [hunk 6]
```diff
@@ -400,7 +400,7 @@ server pool consisting of each of the controller nodes is sufficient.
 Runtime Fabric on VMs / Bare Metal requires a minimum of three nodes in a development environment as shown in the
 following diagram:
 
-image::architecture-development.png[]
+image::architecture-development.png["Diagram of a network development architecture of MuleSoft control plane with controllers, such as internal load balancer, agent, and workers"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/install-prereqs.adoc]:
```suggestion
image::architecture-development.png["Diagram of a network development architecture of the MuleSoft control plane with controllers, such as an internal load balancer, agent, and workers"]
```

File: modules/ROOT/pages/app-versioning.adoc [hunk 7]
```diff
@@ -35,7 +35,7 @@ In the *Config changes* tab on the application's *Settings* page. Runtime Fabric
 +
 The last ten configuration deployments are displayed in order of their respective configuration date. Each configuration in the history has a unique hash to identify it.
 
-image::app-versioning.png[]
+image::app-versioning.png["Configuration page for a runtime fabric application, displaying application file, version, deployment target, runtime version, and replicas"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/app-versioning.adoc]:
```suggestion
image::app-versioning.png["Configuration page for a Runtime Fabric application, displaying the application file, version, deployment target, runtime version, and replicas"]
```

File: modules/ROOT/pages/app-versioning.adoc [hunk 8]
```diff
@@ -52,7 +52,7 @@ The *All changes* tab displays a history of versions created from the configurat
 . Once your changes are made, click *Apply Changes*. +
 At the top of the *Settings* page, a status bar appears.
 +
-image:app-versioning-applying-config.png[]
+image:app-versioning-applying-config.png["Applying configuration status for a Mule app in Runtime Manager UI"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/app-versioning.adoc]:
```suggestion
image:app-versioning-applying-config.png["Applying configuration status for a Mule app in the Runtime Manager UI"]
```

File: modules/ROOT/pages/app-versioning.adoc [hunk 9]
```diff
@@ -74,17 +74,17 @@ This model means there is zero downtime while updates are applied to the applica
 [NOTE]
 Since updates are applied as rolling updates, there will be different application versions running at the same time until the update is completed, so make sure your applications are able to run multiple versions in parallel.
 +
-image::app-versioning-replica-status.png[]
+image::app-versioning-replica-status.png["Replicas Status for a Mule app in Runtime Manager UI"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/app-versioning.adoc]:
```suggestion
image::app-versioning-replica-status.png["Replicas Status for a Mule app in the Runtime Manager UI"]
```

File: modules/ROOT/pages/app-versioning.adoc [hunk 10]
```diff
@@ -74,17 +74,17 @@ This model means there is zero downtime while updates are applied to the applica
...
 +
 Once the configuration is successfully applied and deployed, a message appears at the top of the page.
 +
-image::app-versioning-config-applied-success.png[]
+image::app-versioning-config-applied-success.png["Configuration applied successfully for a Mule app in Runtime Manager UI"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/app-versioning.adoc]:
```suggestion
image::app-versioning-config-applied-success.png["Configuration applied successfully status for a Mule app in the Runtime Manager UI"]
```

File: modules/ROOT/pages/app-versioning.adoc [hunk 11]
```diff
@@ -74,17 +74,17 @@ This model means there is zero downtime while updates are applied to the applica
...
 +
 The application status, configuration identifier, date, and other information is updated in the status box.  The new configuration is displayed at the top of the *Config changes* list, along with a *Last successful* label to indicate that it's the last configuration successfully deployed.
 +
 If the updates to the configuration fail to deploy, you get an `Desired configuration wasn't applied` message, and the *Configuration status* displays an `Update failed` label. The configuration is displayed in the *Config changes* history without a label.
 +
-image::app-versioning-update-failed.png[]
+image::app-versioning-update-failed.png["Configuration failed to apply for a Mule app in Runtime Manager UI"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/app-versioning.adoc]:
```suggestion
image::app-versioning-update-failed.png["Configuration failed to apply status for a Mule app in the Runtime Manager UI"]
```

File: modules/ROOT/pages/index-self-managed.adoc [hunk 12]
```diff
@@ -6,7 +6,7 @@ Runtime Fabric on Self-Managed Kubernetes enables you to deploy Mule application
 
 Runtime Fabric on Self-Managed Kubernetes and its related components run as services on your Amazon EKS, AKS, or GKE environment. The following diagram shows different layers of the architecture.
 
-image::architecture-self-managed.png[]
+image::architecture-self-managed.png["Architectural diagram of Mulesoft managed Runtime Fabric services versus Kubernetes management "]
```
Comment (valkyrie69) [inline modules/ROOT/pages/index-self-managed.adoc]:
```suggestion
image::architecture-self-managed.png["Architectural diagram of a Mulesoft managed Runtime Fabric services versus Kubernetes management "]
```

File: modules/ROOT/pages/index-self-managed.adoc [hunk 13]
```diff
@@ -6,7 +6,7 @@ Runtime Fabric on Self-Managed Kubernetes enables you to deploy Mule application
 
 Runtime Fabric on Self-Managed Kubernetes and its related components run as services on your Amazon EKS, AKS, or GKE environment. The following diagram shows different layers of the architecture.
 
-image::architecture-self-managed.png[]
+image::architecture-self-managed.png["Architectural diagram of a Mulesoft managed Runtime Fabric services versus Kubernetes management "]
```
Comment (dmerlob) [inline modules/ROOT/pages/index-self-managed.adoc]:
```suggestion
image::architecture-self-managed.png["Architectural diagram of a Mulesoft managed Runtime Fabric services versus Kubernetes management"]
```

File: modules/ROOT/pages/index-vm-bare-metal.adoc [hunk 14]
```diff
@@ -28,11 +28,11 @@ Anypoint Runtime Fabric on VMs / Bare Metal supports development and production
 
 The development configuration is intended for testing only. It requires at least one controller and two worker nodes. The controller node runs the internal load balancer and agents used to connect to Anypoint Platform. Communication between the agent and Anypoint Platform is always outbound. Multiple replicas of application can run across workers.
 
-image::architecture-development.png[]
+image::architecture-development.png["Diagram of a network development architecture of the MuleSoft control plane with controllers, such as internal load balancer, agent, and workers"]
```
Comment (dmerlob) [inline modules/ROOT/pages/index-vm-bare-metal.adoc]:
```suggestion
image::architecture-development.png["Diagram of a network development architecture of the MuleSoft control plane with controllers, such as an internal load balancer, agent, and workers"]
```

File: modules/ROOT/pages/index-vm-bare-metal.adoc [hunk 15]
```diff
@@ -46,7 +46,7 @@ Mule applications run on workers. Multiple replicas of applications can run acro
 
 The following diagram shows the general network architecture of Runtime Fabric on VMs / Bare Metal.
 
-image::architecture-network.png[]
+image::architecture-network.png["Diagram of network architecture of Runtime Fabric and TCP load balancer connection with internal load balancers of controllers and workers"]
```
Comment (dmerlob) [inline modules/ROOT/pages/index-vm-bare-metal.adoc]:
```suggestion
image::architecture-network.png["Diagram of the network architecture of Runtime Fabric and TCP load balancer connection with an internal load balancers of controllers and workers"]
```

File: modules/ROOT/pages/index-vm-bare-metal.adoc [hunk 16]
```diff
@@ -62,7 +62,7 @@ Enabled only if one or more dedicated internal load balancer nodes is available
 
 Higher inbound traffic loads require dedicated nodes. To determine if dedicated nodes are needed for your environment, refer to the performance information in xref:deploy-resource-allocation.adoc#internal-load-balancer[Internal Load Balancer].
 
-image::architecture-network2.png[]
+image::architecture-network2.png["Diagram of network architecture of Runtime Fabric and TCP load balancer connection with requests ping"]
```
Comment (dmerlob) [inline modules/ROOT/pages/index-vm-bare-metal.adoc]:
```suggestion
image::architecture-network2.png["Diagram of the network architecture of Runtime Fabric and TCP load balancer connection with requests ping"]
```

File: modules/ROOT/pages/index-vm-bare-metal.adoc (hunk 3 above)
Comment (dmerlob) [inline modules/ROOT/pages/index-vm-bare-metal.adoc]:
```suggestion
image::runtimefabricresponsibility.png["Runtime Fabric Shared Responsibilities"]
```

**Review by dmerlob — DISMISSED** [review]

File: modules/ROOT/pages/app-versioning.adoc [hunk 17]
```diff
@@ -74,17 +74,17 @@ This model means there is zero downtime while updates are applied to the applica
 [NOTE]
 Since updates are applied as rolling updates, there will be different application versions running at the same time until the update is completed, so make sure your applications are able to run multiple versions in parallel.
 +
-image::app-versioning-replica-status.png[]
+image::app-versioning-replica-status.png["Replicas Status for a Mule app in the Runtime Manager UI"]
```
Comment (dmerlob) [inline modules/ROOT/pages/app-versioning.adoc:77]:
```suggestion
image::app-versioning-replica-status.png["Replica status for a Mule app in the Runtime Manager UI"]
```

**Review by dmerlob — APPROVED** [review]

**Pull request merged**